packages/experimental/Avatar/src/NativeAvatar.tsx

```diff
@@ -26,7 +26,8 @@ interface ExportedConstants {
   sizes: { [key in Size]: number };
 }
 
-const ExportedNativeConstants: ExportedConstants = TurboModuleRegistry.get('FRNAvatarConstants') || NativeModules.FRNAvatarViewManager;
+const ExportedNativeConstants: ExportedConstants =
+  TurboModuleRegistry.get('FRNAvatarConstants')?.getConstants() || NativeModules.FRNAvatarViewManager;
```

Comment:
Just curious; do you know what the ` || NativeModules.FRNAvatarViewManager` does for us here? Was it an older means for dealing with accessing the constants from turbo vs native modules?